assignments/callbacks.js

```diff
@@ -1,28 +1,43 @@
 const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
-
-function firstItem(arr, cb) {
-  // firstItem passes the first item of the given array to the callback function.
+const callbacks1 = function(item){
+  console.log(item);
+}
+function firstItem(arr, cb){
+  cb(arr[0])
 }
+//firstArr(items, callbacks1);
+   
+
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  cb(arr.length);
 }
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  cb(arr[arr.length - 1]);
 }
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  cb(x + y);
 }
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  cb(x * y);
 }
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  for (let i = 0; i < list.length; i++){
+    if (list[i] == item){
+      cb(true);
```

Comment:
You know what, we should have added a break or a return in that if statement? I'm sorry I didn't see this sooner.